Gemfile

```diff
@@ -6,11 +6,16 @@ else
   gem 'puppet', :require => false
 end
 
+# Use rake ~> 10.1 for ruby 1.8.7
+if RUBY_VERSION <= '1.8.7'
+  gem 'rake', '~> 10.1'
+end
+
 gem 'metadata-json-lint'
 gem 'puppetlabs_spec_helper', '>= 0.1.0'
 gem 'puppet-lint', '>= 1.0.0'
 gem 'facter', '>= 1.7.0'
-gem 'rspec-puppet', '~>1.0'
+gem 'rspec-puppet', '~> 2.0'
```

Comment:
Ok. I did it without limiting it to a specific version at first but then looked at puppetlabs/puppetdb and they seemed to have limited it to 2.0.

Update incoming.